src/main/java/fr/zcraft/quartzlib/components/i18n/translators/gettext/PluralForms.java

```diff
@@ -101,30 +127,50 @@ public int computePluralForm(long count) {
         // We first try if this script is known
         Function<Long, Integer> function = formsFunctionFromKnownScripts();
 
-        // Else we try two JS engines, and fallback to English rules.
+        // Else we try two JS engines, and fallback to English rules if none of them work.
         if (function == null) {
             function = formsFunctionFromNashorn();
```

Comment:
This fix the issue where the console was spammed with the nashor engine deprecated warning?

Reply:
Not this. The issue was in `POFile`: if you look closely, the `PluralForms` instance was (re)constructed for each analysed token, so for each translation, dozens of times per file, instead on only one time at the end.

This is only a visual optimization.